packager/packager.cc

```diff
@@ -1317,6 +1366,9 @@ void rewritePackageSpec(const core::GlobalState &gs, ast::ParsedFile &package, P
     PackageSpecBodyWalk bodyWalk(info);
     core::Context ctx(gs, core::Symbols::root(), package.file);
     ast::TreeWalk::apply(ctx, bodyWalk, package.tree);
+    if (!info.strictDependenciesLevel) {
```

Comment:
```suggestion
    if (!info.strictDependenciesLevel.has_value()) {
```

Personal preference, feel free to ignore, I like making comparisons against `nullptr` and `nullopt` explicit, instead of relying on the `operator bool` / `operator !` methods to do the conversion implicitly.